shelley/chain-and-ledger/executable-spec/test/Rules/TestDeleg.hs

```diff
@@ -12,10 +13,10 @@ import           Data.Set (Set)
 import qualified Data.Set as Set (singleton, size)
 import           Data.Word (Word64)
 
-import           Hedgehog (Property, TestLimit, forAll, property, withTests, (===))
+import           Hedgehog (Property, TestLimit, forAll, property, withTests)
 
 import           Control.State.Transition.Generator (ofLengthAtLeast, trace)
-import           Control.State.Transition.Trace (sourceSignalTargets)
+import           Control.State.Transition.Trace (STTriple (..), sourceSignalTargets)
```

Comment:
I'd make a practice out of explicitly importing the constructors and accessors, otherwise the `..` kills the point of having explicit imports.